test-utils/src/lib.rs

```diff
@@ -174,7 +174,7 @@ impl Default for FeeConfig {
             state_merkle_tree_rollover: 149,
             nullifier_queue_rollover: 29,
             address_queue_rollover: 181,
-            tip: 1,
+            tip: 5000,
```

Comment:
?

Reply:
will be renamed once https://github.com/Lightprotocol/light-protocol/pull/664 is merged